[14.0][MIG] purchase_isolated_rfq

SurachetS opened the pull request.

kongrattapong committed:
[13.0][ADD] purchase_isolated_rfq

oca-travis committed:
[UPD] Update purchase_isolated_rfq.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] icon.png

OCA-git-bot committed:
pre-commit update

kittiu committed:
[FIX] purchase_isolated_rfq, add uninstall_hook to restore original action domain/context

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
purchase_isolated_rfq 13.0.1.1.0

kittiu commented on purchase_isolated_rfq/tests/test_purchase_isolated_rfq.py:
```diff
@@ -0,0 +1,48 @@
+        self.assertEqual(e.exception.args[0], error_message)
+
+    def test_uninstall_hook(self):
+        """Test if related actions are removed when mass editing
```
Why mentioned about mass editing? Can you validate the descriptoin again.

kittiu commented on purchase_isolated_rfq/tests/test_purchase_isolated_rfq.py:
```diff
@@ -0,0 +1,48 @@
+        self.assertEqual(e.exception.args[0], error_message)
+
+    def test_uninstall_hook(self):
+        """Test if related actions are removed when mass editing
+        record is uninstalled."""
```
```suggestion
        """Test if related actions are reset to state before this module was installed"""
```

kittiu reviewed (commented):
Please rebase and push again, to test with Travis.

kittiu committed:
[13.0][FIX] purchase_isolated_rfq, fix cancel error

OCA-git-bot committed:
purchase_isolated_rfq 13.0.1.1.1

kittiu commented:
@SurachetS please rebase.

kittiu commented on purchase_isolated_rfq/models/purchase_order.py:
```diff
@@ -33,7 +33,6 @@ class PurchaseOrder(models.Model):
         ],
         string="RFQ Status",
         readonly=True,
-        related="state",
```
@SurachetS Why remove this?
Note: I checked form v13, this is intended to be related field with store=False.

Note, we have new field rfq_state, but no value in it. And so, when I create a new RFQ, the state is not displayed.

![image](https://user-images.githubusercontent.com/1973598/105006783-afa7ae80-5a69-11eb-82b8-33b4cbcba04f.png)

kittiu reviewed (approved):
LGTM

dong-z committed:
Added translation using Weblate (Chinese (Simplified))

dong-z committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (12 of 12 strings)

Translation: purchase-workflow-13.0/purchase-workflow-13.0-purchase_isolated_rfq
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-13-0/purchase-workflow-13-0-purchase_isolated_rfq/zh_CN/

dong-z committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (12 of 12 strings)

Translation: purchase-workflow-13.0/purchase-workflow-13.0-purchase_isolated_rfq
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-13-0/purchase-workflow-13-0-purchase_isolated_rfq/zh_CN/

dong-z committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (12 of 12 strings)

Translation: purchase-workflow-13.0/purchase-workflow-13.0-purchase_isolated_rfq
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-13-0/purchase-workflow-13-0-purchase_isolated_rfq/zh_CN/

Saran440 commented:
@SurachetS travis is red. could you fix it?

SurachetS committed:
[IMP] purchase_isolated_rfq: black, isort, prettier

SurachetS committed:
[MIG] purchase_isolated_rfq: Migration to 14.0

Saran440 reviewed (approved):
No major change :+1: 

Please, fix red travis

dreispt commented:
/ocabot merge nobump
